.github/workflows/continuous-integration-workflow.yml

```diff
@@ -36,4 +36,4 @@ jobs:
         python -m pip install --upgrade pip
         pip install -r requirements.txt
     - name: Lint with pylint
-      run: export FLASK_APP_ENV=test; pylint src/ script/ --disable=C0116,E1101,R0903,C0115,C0103,C0301,C0114,W0621,W0511,E0611,R0801
+      run: export FLASK_APP_ENV=test; pylint script/ tests/ notification_delivery/ external_integrations/ --disable=C0116,E1101,R0903,C0115,C0103,C0301,C0114,W0621,W0511,E0611,R0801
```

Comment:
Just curious if there's a way to run pylint on the entire source tree. If not, this is fine.

Reply:
I looked into it and found that you can't run "pylint . " unless the directory you're in is a package itself (has __init__.py) which is not the case for this project's root directory. The other option I found was to use " pylint *.py **/*.py ", but this gave me the same error that April got (that *.py was not found) when there were no .py files in the root of the repository.